generators/common/templates/README.md.ejs

```diff
@@ -247,14 +247,18 @@ Refer to [Using JHipster in production][] for more details.
 
 To package your application as a war in order to deploy it to an application server, run:
 <% if (buildTool === 'maven') { %>
-    ./mvnw -Pprod,war clean package<% } %><% if (buildTool === 'gradle') { %>
+    ./mvnw -Pprod,war clean verify<% } %><% if (buildTool === 'gradle') { %>
     ./gradlew -Pprod clean bootWar<% } %>
 
 ## Testing
 
 To launch your application's tests, run:
 
-    <% if (buildTool === 'maven') { %>./mvnw clean test<% } else { %>./gradlew test<% } %>
+    <%_ if (buildTool === 'maven') { _%>
+    ./mvnw test integration-test
```

Comment:
Unless I'm misunderstanding something, I would just use `verify` (it also includes `test`) rather than `integration-test`, according to https://maven.apache.org/guides/introduction/introduction-to-the-lifecycle.html : 
>  If integration-test were to be called from the command line, no reports are generated. Worse is that the integration test container environment is left in a hanging state; the Tomcat webserver or Docker instance is left running, and Maven may not even terminate by itself.

Reply:
Ok thx, @murdos -> I'm doing another PR to change this

Reply:
The only minor issue I saw is `packaging` the application and `launching` tests is the same command now. So the information is here twice and is similar to Gradle
But it's really minor